package.json

```diff
@@ -21,6 +21,7 @@
     "build": "lerna run build:viewer --stream",
     "build:dev": "lerna run build:dev --stream",
     "build:ci": "lerna run build:viewer:ci --stream",
+    "build:static": "lerna run build:static --stream && cp -rf ./platform/viewer/dist/* ~/ohif/.",
```

Comment:
Don't want to include this line - suggest you run:
set APP_CONFIG=config/local_static.js
yarn start
which will automatically rebuild